citi-onboarding/acervusej-onepage
# Feat/sendemail frontend

Things I've done:

-  Store the inputs of the contact section and store them in a state
- Make a function to send this data in the necessary fields of an email message, based on the backend
- The function will be called when the send button is clicked, and the fields will be cleared when the message is sent
- Adjust the css of the contact section

How to test it:

- First download the node modules on the client and the server. If you have already done that, then:
- Change the values on .env.
- Go to client on the terminal (cd client) and put npm run start.
- On the browser write http://localhost:3000/.
- See if the contact section works both in mobile and desktop view.
- Try to contact the acervus (try to break it by putting spaces with nothing on it).
- See if the email arrived with everything in the right place
